scripts/base/frameworks/cluster/pools.zeek

```diff
@@ -25,7 +25,7 @@ export {
 		## A topic string that can be used to reach all nodes within a pool.
 		topic: string &default = "";
```

Comment:
Have you considered un-defaulting this one, too? I'm not versed with the topics. Would not changing this and subscribing to an empty topic be a wild-card match, or even an error?

Reply:
I did! I was torn. I do think it would be better not to default it, and in terms of our current `PoolSpec` use it's just as safe/risky to change it as `node_type`. But the semantics of empty topics aren't clear to me either, so I didn't go near it.

I did some digging today and the details might only start to matter down in CAF — I couldn't find explicit treatment or objection to empty topics at the higher levels. But it's clear that the topic is not meant to remain empty, so I come out in favor of un-defaulting it.